### DISCUSSION TITLE
Fan Controls

### DISCUSSION THREAD

**Discussion by Oblivon23** [seerge/g-helper#3493] [Q&A]
So I've got the Strix G15 2022 (6900HX+3070 Ti)

I've just realized but the fan controls seems to be inverted on g-helper?

Adjusting the GPU fan changes the CPU fan, the same for the CPU Fan acting as thr gpu fan. Is this the supposed behavior?

**Comment by seerge** [comment]
@Oblivon23 hello, G-Helper is not controlling fans, neither does AC - they are controlled by BIOS / firmware.

As far as I remember G15 had notoriously bugged bios, that Asus was fixing like few times with multiple releases. Do you use latest one ? :)

**Reply by Oblivon23** [reply]
Yes, I'm using version 327. I've been wondering why my CPU temps seem the same regardless of my fan profile on the CPU Fan

**Reply by seerge** [reply]
@Oblivon23 if firmware is not following fan curve precisely, then it's probably programmed like that :) If you need custom fan behavior by some reason you can try [Experimental build with a manual fan control](https://github.com/seerge/g-helper/discussions/2272)
